hackathon/templates/hackathon/includes/hackathon_card.html

```diff
@@ -1,11 +1,48 @@
 <article class="card shadow">
-	<div class="card-body">
-		<h5 class="p-orange card-title">{{ hackathon.display_name }}</h5>
-		<h6 class="card-subtitle mb-2 text-muted">{{ hackathon.start_date }} - {{ hackathon.end_date }}</h6>
-		<p class="card-text">{{ hackathon.description }}</p>
-        {% if hackathon.organiser %}
-		<p>Organiser: <a href="#" class="p-blue card-link">{{ hackathon.organiser }}</a></p>
+  <div class="card-body">
+    <h5 class="p-orange card-title">{{ hackathon.display_name }}</h5>
+    <h6 class="card-subtitle mb-2 text-muted">{{ hackathon.start_date }} - {{ hackathon.end_date }}</h6>
+    <p class="card-text">{{ hackathon.description }}</p>
+    {% if hackathon.start_date < now %}
+      Less than
```

Comment:
What does that mean? Is this for testing purposes? If it is please remove it.

Reply:
Yep, it was for testing purposes to figure out how I could make the date logic work in the modal. I just forgot to take it out! I've removed it now and it should be visible when I push the changes to the existing PR.